backend.js

```diff
@@ -11,6 +11,18 @@ var tables = [];
 
 var waitList = [];
 
+app.get("/", function(req, res) {
+    res.sendFile(path.join(__dirname, "home.html"));
+  });
+
+  app.get("/api/tables", function(req, res) {
+    res.sendFile(path.join(__dirname, "tables.html"));
+  });
+
+  app.get("api/reserve", function(req, res) {
+    res.sendFile(path.join(__dirname, "reserve.html"));
```

Comment:
Same as above.  This path should just be "/reserve" to take the user to the reserve.html page.  The path "/api/waitlist" will take a user to an empty page showing the waitlist array